src/soho/bar/soho-bar.component.ts

```diff
@@ -225,4 +225,16 @@ export class SohoBarComponent implements AfterViewInit, OnDestroy {
     }
   }
 
+  public setSelected(selected: SohoBarSelected) {
+    this.bar.setSelected(selected);
+  }
```

Comment:
This is really what soho bar.js takes? a SohoBarSelect?

Reply:
Each of the Charts type has its own definition file so it seemed that was the right way to go. Some chart take different params